app/models/spree_multi_vendor/spree/order_decorator.rb

```diff
@@ -66,7 +66,7 @@ def vendor_commission(vendor)
   end
 
   # money methods
-  METHOD_NAMES = %w[
+  METHOD_NAMES ||= %w[
```

Comment:
JFYI:

What I thought of as a second candidate: Just using ivar. 


```diff
diff --git a/app/models/spree_multi_vendor/spree/order_decorator.rb b/app/models/spree_multi_vendor/spree/order_decorator.rb
index c5f9f5f..2415d7b 100644
--- a/app/models/spree_multi_vendor/spree/order_decorator.rb
+++ b/app/models/spree_multi_vendor/spree/order_decorator.rb
@@ -66,12 +66,12 @@ module SpreeMultiVendor::Spree::OrderDecorator
   end
 
   # money methods
-  METHOD_NAMES = %w[
+  method_names = %w[
     total ship_total subtotal included_tax_total additional_tax_total promo_total
     pre_tax_item_amount pre_tax_ship_amount pre_tax_total commission
-  ].freeze
+  ]
 
-  METHOD_NAMES.each do |method_name|
+  method_names.each do |method_name|
     define_method("display_vendor_#{method_name}") do |vendor|
       Spree::Money.new(send("vendor_#{method_name}", vendor), { currency: currency })
     end
```

Reply:
Yes, I think keeping it as a constant is better IMO